packages/fixie/src/agent.ts

```diff
@@ -317,30 +343,214 @@ export class FixieAgent {
         config.public
       );
     }
+    return agent;
+  }
+
+  static spawnAgentProcess(agentPath: string, port: number): ChildProcess {
+    term(`🌱 Starting local agent process on port ${port}...\n`);
+    if (!fs.existsSync(`${agentPath}/dist/index.js`)) {
+      throw Error(`No dist/index.js found in ${agentPath}. Do you need to build your agent code first?`);
+    }
```

Comment:
```suggestion
    const pathToCheck = path.resolve(path.join(agentPath, 'dist', 'index.js`));
    if (!fs.existsSync(pathToCheck)) {
      throw Error(`Your agent was not found at ${pathToCheck}. Do you need to build your agent code first?`);
    }
```

We need to use `path.join` instead of hardcoding `/` if we want Windows compat.